packages/ui/scripts/release-script.ts

```diff
@@ -0,0 +1,176 @@
+import {
+  bumpPackageJsonVersion,
+  executeAndLog,
+  getPackageJsonVersion,
+  bumpGithubTemplateVersions,
+  bumpVersionInGenerators,
+  executeCommand, getCommitHash,
+} from '../build/utils'
+
+import yargs from 'yargs/yargs'
+import semver from 'semver'
+
+const { hideBin } = require('yargs/helpers')
+
+export type ReleaseType = 'monthly' | 'weekly' | 'next' | 'experimental'
+
+export type ReleaseConfig = {
+  version: string, // 1.2.3
+  distTag: 'latest' | 'next' | 'experimental',
+  gitTag: string, // v1.2.3
+  branch: string, // 'develop'
+  requiredBranch: string // 'develop'
+  commit: string, // '12345678'
+  shouldCommit: boolean,
+}
+
+const argv: {
+  releaseType: ReleaseType,
+  semverReleaseType: string,
+  dryRun: boolean
+} = yargs(hideBin(process.argv))
+  .option('releaseType', {
+    description: 'Internal release type',
+    choices: ['monthly', 'weekly', 'next', 'experimental'],
```

Comment:
```suggestion
    choices: ['major', 'minor', 'patch', 'next', 'experimental'],
```

How about to make it a big easier?

Reply:
I don't think it's a good idea to keep semver release typing for vuestic release process which has nothing to do with semver.